FirebaseRemoteConfig/Sources/RCNConfigRealtime.m

```diff
@@ -103,6 +99,11 @@ @implementation RCNConfigRealtime {
   RCNConfigSettings *_settings;
   FIROptions *_options;
   NSString *_namespace;
+  NSInteger gMaxRetryCount;
+  NSInteger gRetrySeconds;
```

Comment:
fyi, for all the ivar read, you can use dispatch_sync to guard it with a serial queue so it can returns the value right away.

I can't see the whole code base so just in case if there's reading from those ivars make sure you also guard it with serial queue.

Reply:
Yeah I made sure that all of these vars are guarded with serial queues